srcpkgs/wayfire/template

```diff
@@ -1,23 +1,26 @@
 # Template file for 'wayfire'
 pkgname=wayfire
-version=0.5.0
-revision=2
-_utils_commit=f9b5eba437a04a0d1fb9f00a0fdb88c12b9f6b27
+version=0.6.0
+revision=1
+_utils_commit=f45641beef46babdc8f1b8d18a924e72beaf8ee6
+_touch_commit=b1075c54a280f913edc26b9757262f4f9d6b62b0
 build_style=meson
 hostmakedepends="pkg-config wayland-devel"
 makedepends="wf-config-devel wlroots-devel cairo-devel
  $(vopt_if image 'libjpeg-turbo-devel libpng-devel')"
-depends="xorg-server-xwayland $(vopt_if elogind elogind)"
+depends="xorg-server-xwayland $(vopt_if elogind elogind) $(vopt_if seatd seatd)"
 short_desc="3D wayland compositor"
 maintainer="Érico Nogueira <ericonr@disroot.org>"
 license="MIT"
 homepage="https://wayfire.org"
 distfiles="https://github.com/WayfireWM/wayfire/archive/${version}.tar.gz
- https://github.com/WayfireWM/wf-utils/archive/${_utils_commit}.tar.gz"
-checksum="24c1a2c963dac5af762f87cd024bc3dd736ec9a28a6735d357a05e8f6502e8aa
- 5c3e8bfefd74083a2548b6a95a070000cf73591bfe78335413da5c7fb82340cb"
+ https://github.com/WayfireWM/wf-utils/archive/${_utils_commit}.tar.gz
+ https://github.com/WayfireWM/wf-touch/archive/${_touch_commit}.tar.gz"
+checksum="9c2bf92e6aecc9b800b87e4c086ff7a275393f7315501b8c39196baf9d245b33
+ d172f8c21e0bac01e4116cd957fb0159c5cb39ddfdce897beb0d9c753796d5f1
+ 2b22e03d3a522baeff5798f630ffe5aa95899fd3233b291527503af5fd3e30be"
 
-build_options="elogind image"
+build_options="elogind seatd image"
```

Comment:
Could this option be renamed to something like `seat_manager`?

Reply:
If the option were called `setuid`, you could just drop the two `vopt_if` conditions in `depends` and use `if [ "$build_option_setuid" ]` to gate the `chmod` in `post_install`.

Reply:
And then the compositors won't depend directly on `seatd` or `elogind`. I like this solution.

Reply:
Should be fixed in all instances.